### PR TITLE
JIRA Fixes

### PR DESCRIPTION
**Pull request by Minoli**

### CODE REVIEW

File: jaggeryapps/admin/js/custom/custom.js [hunk 1]
```diff
@@ -598,3 +602,6 @@ function checkAppList(cookie,userName) {
 
     return spList;
 }
+$(document).ready(function () {
```
Comment (jeradrutnam) [inline jaggeryapps/admin/js/custom/custom.js]: Why are we having an empty document ready function here?

File: jaggeryapps/admin/serviceproviders/addsp.jag [hunk 2]
```diff
@@ -198,90 +222,12 @@ include("../templates/footer.jag");
     $(document).on('click', '.cloud-app-type', function(){
```
Comment (jeradrutnam) [inline jaggeryapps/admin/serviceproviders/addsp.jag]: Shall we move this script block to a js file rather keeping in the jag file itself?

File: jaggeryapps/admin/js/custom/custom.js (hunk 1 above)
Comment (Minoli) [inline jaggeryapps/admin/js/custom/custom.js]: This is been removed

**Comment by malithie** [comment]
@Minoli, can you please update the PR with suggested improvements. We need to get this merged soon

**Review by malithie — APPROVED** [review]
Review comments will be fixed via a new PR. 
@Minoli, please refer the new PR from here once you fixed it.

**Pull request merged**